frontend/src/components/pages/Scheduling/DonationInformation.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button, Container } from "@chakra-ui/react";
+import React from "react";
+import { NavigationProps, SetForm } from "react-hooks-helper";
+import { useHistory } from "react-router-dom";
+
+import { SchedulingFormProps } from "./types";
+
+const DonationInformation = ({
+  formValues,
+  setForm,
+  navigation,
+}: {
+  navigation: NavigationProps;
+  formValues: SchedulingFormProps;
+  setForm: SetForm;
+}) => {
+  const { next } = navigation;
```

Comment:
The figma mocks also have a back button, should we add it in this PR?
```suggestion
  const { previous, next } = navigation;
  <Button onClick={previous}>Back</Button>
```